src/files/file-input/FileInput.js

```diff
@@ -20,7 +19,7 @@ const AddButton = translate('files')(({ progress = null, t, tReady, ...props })
   }, ['Button f6 relative transition-all sans-serif dib v-mid fw5 nowrap lh-copy bn br1 pa2 focus-outline'])
 
   return (
-    <button disabled={sending} className={cls} style={{ width: '144px' }} {...props}>
+    <button disabled={sending} className={cls} style={{ width: '120px' }} {...props}>
```

Comment:
huh!? this never used our <Button> component. Not related to your PR @fsdiogo but could you take a look at  swapping this out for <Button> , to make things more consistent.